### PR TITLE
Add draft of generic ZoomControl component [wip] 

### PR DESCRIPTION
**Pull request by pablopareja**

### CODE REVIEW

File: src/components/ui/ZoomControl/index.jsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    // BINDINGS
+    this.triggerZoomIn = this.triggerZoomIn.bind(this);
+    this.triggerZoomOut = this.triggerZoomOut.bind(this);
+  }
+
+  triggerZoomIn() {
+    const newValue = this.state.value + 1;
+    if (newValue <= this.props.maxValue) {
+      this.setState({ value: newValue });
+      this.props.onZoomIn(this.state.value);
```
Comment (mbarrenechea) [inline src/components/ui/ZoomControl/index.jsx]: Be careful, `this.setState` is asynchronous so `this.props.onZoomIn(this.state.value);` should be on the callback of `this.setState`. The same with the ZoomOut

**Review by mbarrenechea — CHANGES_REQUESTED** [review]

File: src/components/ui/ZoomControl/index.jsx (hunk 1 above)
Comment (pablopareja) [reply src/components/ui/ZoomControl/index.jsx]: You're right, thanks! :+1:

**Comment by pablopareja** [comment]
The ZoomControl component is finished and has been added to the Planet Pulse page. We're now waiting for the styles to be defined by the design team.

**Comment by pablopareja** [comment]
The styles have been defined and applied to ZoomControl so the component should be finished in its current state. 
@davidsingal Could you have a look at it? Thanks!

**Comment by davidsingal** [comment]
It fails a bit when you zoom using scroll mouse and then you will click on zoom buttons

**Comment by pablopareja** [comment]
It should be fixed now

**Pull request merged**